packages/block-editor/src/utils/transform-styles/ast/parse.js

```diff
@@ -240,7 +240,7 @@ export default function ( css, options ) {
 	function declaration() {
 		const pos = position();
 
-		// prop
+		// prop.
```

Comment:
It doesn't start with an upper case. Is it because it's also a variable in the scope? Anyway, let's be honest that this comment is redundant similar to other single words in the same file.